packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
```

Comment:
I wish logrotate had a "rotate <size>" option in addition to the "rotate <count>" option.